src/renderer/features/networks/NetworkFilter/model/network-filter.ts

```diff
@@ -0,0 +1,38 @@
+import { combine, createEvent, createStore, sample } from 'effector';
+
+import { includes } from '@shared/lib/utils';
+import { networkModel } from '@entities/network';
+
+const componentMounted = createEvent();
+
+const $filterQuery = createStore<string>('');
+const queryChanged = createEvent<string>();
+const queryReset = createEvent();
+
+$filterQuery.on(queryChanged, (_, query) => query).reset(queryReset);
+
+sample({
+  clock: componentMounted,
+  target: queryReset,
+});
```

Comment:
`reinit` is a built-in reset event
`formInitiated` is preferable name, in future this event will be called even before React mounts the component 😉  (see [reflect](https://github.com/effector/reflect))
```suggestion
sample({
  clock: formInitiated,
  target: $filterQuery.reinit,
});
```

Reply:
Contact form was the initial place where effector was integrated, so it contains outdated code style 😄